unstructured/ingest/v2/examples/example_pinecone.py

```diff
@@ -0,0 +1,49 @@
+from pathlib import Path
+
+from unstructured.ingest.v2.interfaces import ProcessorConfig
+from unstructured.ingest.v2.logger import logger
+from unstructured.ingest.v2.pipeline.pipeline import Pipeline
+from unstructured.ingest.v2.processes.chunker import ChunkerConfig
+from unstructured.ingest.v2.processes.connectors.local import (
+    LocalConnectionConfig,
+    LocalDownloaderConfig,
+    LocalIndexerConfig,
+)
+from unstructured.ingest.v2.processes.connectors.pinecone import (
+    PineconeAccessConfig,
+    PineconeConnectionConfig,
+    PineconeUploaderConfig,
+    PineconeUploadStagerConfig,
+)
+from unstructured.ingest.v2.processes.embedder import EmbedderConfig
+from unstructured.ingest.v2.processes.partitioner import PartitionerConfig
+
+base_path = Path(__file__).parent.parent.parent.parent.parent
+docs_path = base_path / "example-docs"
+work_dir = base_path / "tmp_ingest"
+output_path = work_dir / "output"
+download_path = work_dir / "download"
+
+if __name__ == "__main__":
+    logger.info(f"Writing all content in: {work_dir.resolve()}")
+    Pipeline.from_configs(
+        context=ProcessorConfig(work_dir=str(work_dir.resolve())),
+        indexer_config=LocalIndexerConfig(input_path=str(docs_path.resolve()) + "/multisimple/"),
+        downloader_config=LocalDownloaderConfig(download_dir=download_path),
+        source_connection_config=LocalConnectionConfig(),
+        partitioner_config=PartitionerConfig(strategy="fast"),
+        chunker_config=ChunkerConfig(
+            chunking_strategy="by_title",
+            chunk_include_orig_elements=False,
+            chunk_max_characters=1500,
+            chunk_multipage_sections=True,
+        ),
+        embedder_config=EmbedderConfig(embedding_provider="langchain-huggingface"),
+        destination_connection_config=PineconeConnectionConfig(
+            access_config=PineconeAccessConfig(api_key="0e8555a2-b944-4da4-837e-03b1202e00c7"),
```

Comment:
have rotated it, but reminder in the future don't use real values for keys. I typically don't even put in anything that even looks like a key but instead do something like `"<YOUR_PINECONE_API_KEY_HERE>"` to make it very clear and avoid.

Reply:
maybe worth just updating this to avoid that confusion?

Reply:
alternatively, if you want a working example script, could have it pull from an env var.

Reply:
All agreed! Sorry about this